File: android_labs/mplab08/app/models/ContactModel.java
package shv.fit.bstu.mp_lab06.models;

import java.io.Serializable;

public class ContactModel implements Serializable {
    public String name;
    public String email;
    public String location;
    public String phone;
    public String url;
    public String cimg;

    public ContactModel(){

    }

    public ContactModel(String name, String email, String location, String phone, String url, String cimg){
        this.name = name;
        this.email = email;
        this.location = location;
        this.phone = phone;
        this.url = url;
        this.cimg = cimg;
    }
}

File: pi-lab08a/pi-lab08a-api/Models/ContactModel.cs
namespace pi_lab08a_api.Models
{
    public class ContactModel
    {
        public int id { get; set; }
        public string name { get; set; }
        public string number { get; set; }

        public ContactModel() { }
        public ContactModel(int id, string name, string number)
        {
            this.id = id;
            this.name = name;
            this.number = number;
        }
        public ContactModel(string name, string number)
        {
            this.name = name;
            this.number = number;
        }
    }
}
